test/turbopack-build-tests-manifest.json

```diff
@@ -9056,13 +9056,13 @@
         "Edge runtime code with imports Middleware importing unused 3rd party module production mode does not build and reports module not found error",
         "Edge runtime code with imports Middleware importing unused 3rd party module throws not-found module error in dev at runtime and highlights the faulty line",
         "Edge runtime code with imports Middleware importing unused node.js module does not throw in dev at runtime",
-        "Edge runtime code with imports Middleware statically importing node.js module throws unsupported module error in dev at runtime and highlights the faulty line",
```

Comment:
@timneutkens I disabled these tests because these tests are mainly about linting  and it does not look like a bug of tree shaking. I tried disabling tree shaking by using `SplitResult::Failed` for all inputs, but it still failed.

AFAIK, `NextEdgeUnsupportedModuleReplacer` is interfering with `next_edge_node_api_assert`, which both concern node-js-specific modules.


I prefer to investigate this issue separately and get tree-shaking PR reviewed first. Thoughts?